src/styles/stopwatch.scss

```diff
@@ -0,0 +1,66 @@
+* {
+  margin: 0;
+}
+
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.stopwatch {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+
+  height: $stopwatchRadius;
+  width: $stopwatchRadius;
+  border-radius: $circle;
+  border: 1vmin dotted black;
+
+  &__minutes {
+    height: 20vmin;
+    width: 3vmin;
+    background-color: #0700ff;
+
+    @include centererd-hands;
+
+    transform-origin: bottom;
+    animation: move 3600s steps(60) infinite;
+  }
+
+  &__seconds {
+    height: 38vmin;
+    width: 1.5vmin;
+    background-color: #2c8000;
+
+    @include centererd-hands;
```

Comment:
Same as the previous comment, the mixin name `centererd-hands` appears to be a typo and should be corrected to `centered-hands`.